state-chain/pallets/cf-broadcast/src/tests.rs

```diff
@@ -917,21 +953,23 @@ fn witness_broadcast(tx_out_id: [u8; 4]) {
 
 #[test]
 fn timed_out_broadcaster_are_reported() {
```

Comment:
while we're here...
```suggestion
fn timed_out_broadcasters_are_reported() {
```

Reply:
not strictly part of this PR so feel free to leave out (and maybe include in a separate PR) - but this test says that we report them... but then stops short of actually testing that - which we can fix, we do:
```
MockOffenceReporter::assert_reported(
			PalletOffence::FailedToBroadcastTransaction,
			failed_authorities,
		);
``` 
elsewhere in these tests - no reason we couldn't here.

Reply:
Simply adding this into the last `then_execute_at_next_block` fails though. The failed broadcasters are appearently only reported once the transaction itself has been successful (in `egress_success`). So if you want to add this I guess there should be some steps more happening, and would be better in a new PR.

Reply:
yeah makes sense to do separately